src/fungible_token/state.rs

```diff
@@ -0,0 +1,57 @@
+use codec::{Decode, Encode};
+use gstd::{prelude::*, ActorId};
+use scale_info::TypeInfo;
+
+#[derive(Debug, Default)]
+pub struct FTState {
+    // Token name
```

Comment:
If these comments are intended to be doc-comments, need to use doc-comments syntax.
```suggestion
    /// Token name.
```